code/src/components/Nav.js

```diff
@@ -0,0 +1,72 @@
+import React, { useRef } from "react";
+import { Link } from "react-router-dom";
+
+import useDetectOutsideClick from "./UseDetectOutsideClick";
+import "./nav.css";
+
+// I wanted to do a dropdown, found this "dropdownRef"-code below and tryed it out!
+
+const Nav = () => {
+  const dropdownRef = useRef(null);
+  const [isActive, setIsActive] = useDetectOutsideClick(dropdownRef, false);
+  const onClick = () => setIsActive(!isActive);
```

Comment:
A bit more information in function name would be very nice. At least let other developers when will this function be executed : `const onButtonClick = () => { ... }`